wire-grpc-server/build.gradle.kts

```diff
@@ -21,7 +22,13 @@ dependencies {
   implementation(libs.grpc.stub) {
     exclude(group = "com.google.guava", module = "guava")
   }
-  implementation("com.google.guava:guava:21.0")
+  implementation(libs.checker.qual)
+  implementation(libs.guava){
+    attributes {
+      // We help Gradle pick between the jre and android of Guava.
```

Comment:
Is it needed? Does Gradle resolve it to the Android version by default?

Reply:
It doesn't resolve it at all actually so I had to tell him

![image](https://github.com/square/wire/assets/1767669/345fc4d1-743a-4293-85a1-5c3fc11a9216)

And others are seeing it: https://github.com/google/guava/issues/6612

Reply:
I talked with @martinbonnin and we cannot repro without the attribute so I tried a commit without any of them.

Reply:
🤞